lib/teamcity/sqlServerCe/sqlServerCe_installation.txt

```diff
@@ -0,0 +1,15 @@
+
+Installation steps for local testing:
+
+1.  Create a project referencing System.Data.SqlServerCe.dll
+2.  Put following code in it, changing database file name and path as appropriate:
+	using (var en = new SqlCeEngine("DataSource=\"C:\\SqlCeData\\NHibernate.sdf\""))
+	{
+		en.CreateDatabase();
+	}
+3.  Run it.
```

Comment:
Indeed, I have taken it from there. So, another reason to not try putting that in the test project.